### PR TITLE
06 view comments list

### PR DESCRIPTION
**Pull request by ZeusLGR**

### CODE REVIEW

File: src/components/CommentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import * as api from '../utils/api'
+import { useEffect, useState } from 'react';
+import moment from 'moment';
+moment().format();
```
Comment (mallionaire) [inline src/components/CommentCard.jsx:4]: Is this doing anything? 
Or is it a leftover?

File: src/components/Home.jsx [hunk 2]
```diff
@@ -7,26 +7,16 @@ import ArticleCard from './ArticleCard';
...
-    const [popularCodingArticle, setPopularCodingArticle] = useState([]);
-    const [popularFootballArticle, setPopularFootballArticle] = useState([]);
-    const [popularCookingArticle, setPopularCookingArticle] = useState([]);
+   
     const [popularArticles, setPopularArticles] = useState([]);
 
     useEffect(() => {
-        api.fetchPopularCodingArticles().then((data) => {
-            setPopularCodingArticle([data.articles[0]]);
+        api.fetchPopularArticles().then((pops) => {
```
Comment (mallionaire) [inline src/components/Home.jsx:14]: `pops` is my new favourite variable name 😂

**Review by mallionaire — APPROVED** [review]
Left you a couple of comments but this is looking wonderful - very clear to read, sensible layout and I think the simplistic look - works very well - I can tell you've put a lot of effort into that styling and its great! 

My only criticism is that you're not following guidance for using semantic html elements, there are a LOT of divs that have crept into your code! 
You might want to read [this guidance as a quick reminder](https://www.w3schools.com/html/html5_semantic_elements.asp)

File: src/components/Home.jsx (hunk 2 above)
Comment (ZeusLGR) [reply src/components/Home.jsx:14]: Hahaha, thanks so much for the feedback! I'll be more mindful of using semantic html elements as I go, for sure.

**Pull request merged**